### DISCUSSION TITLE
V3: config option isn't wired up

### DISCUSSION THREAD

**Discussion by deanzod** [livewire/livewire#5845] [7. Bugs (archived)]
in config/livewire.php changing this to false does nothing. Also after searching the source I can't see it being used at all anywhere

```php
'navigate' => [
        'show_progress_bar' => true,
    ],
```

**Comment by joshhanley** [comment]
@deanzod thanks for reporting! Yeah it possibly got missed. Any chance you could submit a PR with a fix?

**Reply by PhiloNL** [reply]
Submitted a PR, but it requires some changes on the navigate plugin :) https://github.com/livewire/livewire/pull/5874

**Reply by joshhanley** [reply]
@PhiloNL thanks! I will get Caleb to have a look at that one 🙂

**Reply by addtobag** [reply]
Is this the same for 'pagination_theme' => 'bootstrap', doesn't seem to apply when setting in the config file.

**Reply by joshhanley** [reply]
@addtobag I think that's more related to #5851. Can you see my comment over there?

**Reply by addtobag** [reply]
Hi Josh,

protected $paginationTheme = 'bootstrap';

I'm finding if i don't declare the above in the livewire component then the pagination loses all bootstrap styling. I would have expected that setting 'pagination_theme' => 'bootstrap' in the livewire v3 config would ensure you don't need to set it in the component.

Thanks

**Reply by addtobag** [reply]
Nevermind, looks like it's been requested already #5810 

Thanks
